### PR TITLE
Added popup for smaller screens

### PR DESCRIPTION
**Pull request by ujjwxl**
Added popup to inform that game is for PC users for the present

Screenshot (1789)

### CODE REVIEW

**Comment by Ramzi-Abidi** [comment]
@ujjwxl appreciate your time, I think popup is not the best approach here 😅

It kind of ruined the UI, I think [react-alert](https://mui.com/material-ui/react-alert/) will be a better option !

Try using "**warning**" alert from top of the page. 

**Example of alerts:**

image

Sorry for the inconvenience! 😅

File: src/views/Home.tsx
Comment (Ramzi-Abidi) [inline src/views/Home.tsx:1]: @ujjwxl, appreciate your time. I don't think the popup is the best approach here, 😅 it kind of ruined the UI. I believe using [react-alert](https://mui.com/material-ui/react-alert/) would be a much better option!

Consider using the "warning" alert from the top of the page.

Sorry again for the inconvenience! 😅

**Review by Ramzi-Abidi — CHANGES_REQUESTED** [review]
@ujjwxl, appreciate your time. I don't think the popup is the best approach here, 😅 it kind of ruined the UI. I believe using [react-alert](https://mui.com/material-ui/react-alert/) would be a much better option!

Consider using the "warning" alert from the top of the page.

Sorry again for the inconvenience! 😅

**Comment by ujjwxl** [comment]
Alright, will be working on these.

**Pull request closed without merging**